### ISSUE TITLE
Export archive fails due to "Bundle only contains bitcode-marker" error

### ISSUE THREAD

**Issue by sungjin0213** [facebook/SocketRocket#603]
Checked out the library with Carthage, and it's failing to archive with following error message.

```
error: exportArchive: Failed to verify bitcode in SocketRocket.framework/SocketRocket:
error: Bundle only contains bitcode-marker 
```

It seems to be bitcode compiling issue with Carthage that can be resolved by manually setting bitcode modes in the SocketRocket framework: https://github.com/thoughtbot/Argo/pull/260

Would you add these, so that projects that uses Carthage can use SocketRocket framework?

Environment:
- Carthage version: 0.31.2
- Using Cartfile `github "facebook/SocketRocket" "0.4.2"` to get SocketRocket library
- Xcode 10.1

**Comment by kernchen** [comment]
Experiencing the same issue, but when trying to Distribute Ad-Hoc build from Xcode Archiver. The workaround for this was to add a user defined build setting:

Under “Build Settings” add a new user defined setting by pressing the + sign and then select “Add User-Defined Setting”, the setting is: "BITCODE_GENERATION_MODE" with the value “bitcode”

Then needed to manually rebuild SocketRocket with carthage:

`carthage build SocketRocket --platform ios --no-use-binaries`

**Comment by sungjin0213** [comment]
@kernchen thank you for reaching out. This worked except that we had to do a bit more modification in SocketRocket project in order for it to work since there was some compile error. 

I will remain this case open until someone from SocketRocket team resolves this issue.

**Comment by kernchen** [comment]
@sungjin0213, yes you are right, I actually had to open the project and there was some warning about unused return values. I added a (void) in from of the two lines to fix this temporarily.

Glad it helped.
